runtime/amplitude/src/lib.rs

```diff
@@ -568,6 +550,8 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type XcmpMessageHandler = XcmpQueue;
 	type ReservedXcmpWeight = ReservedXcmpWeight;
 	type CheckAssociatedRelayNumber = RelayNumberStrictlyIncreases;
+	#[cfg(feature = "std")]
+	type ConsensusHook = cumulus_pallet_parachain_system::consensus_hook::ExpectParentIncluded;
```

Comment:
This is actually an odd one. Now, this type is only supposed to be required when [this flag](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.1.0/cumulus/pallets/parachain-system/src/lib.rs#L238) is used with the pallet. For some reason that `parameterized-consensus-hook` flag gets enabled on the integration tests, throwing a compilation error if we don't have this type in our runtimes.

Therefore I added `"cumulus-pallet-parachain-system/parameterized-consensus-hook"` to the `std` array for the runtimes. 

We really do not want this new type since we do not want to add any custom hook. My assumption is that this is not relevant for the runtimes on test, benchmarks or any `std` environment.

Reply:
I see, makes sense to me.